### PR TITLE
update _is_deleted to null when merge insert/update events

### PR DESCRIPTION
**Pull request by seanzhougoogle**

### CODE REVIEW

File: src/main/java/io/cdap/delta/bigquery/BigQueryEventConsumer.java [hunk 1]
```diff
@@ -1101,7 +1101,7 @@ static String createMergeQuery(TableId targetTableId, List<String> primaryKeys,
         .filter(predicate)
         .map(Schema.Field::getName)
         .map(name -> String.format("%s = D.%s", name, name))
-        .collect(Collectors.joining(", ")) + "\n" +
+        .collect(Collectors.joining(", ")) +  ", " + Constants.IS_DELETED + " = null\n" +
```
Comment (sagarkapare) [inline src/main/java/io/cdap/delta/bigquery/BigQueryEventConsumer.java:1107]: Do we need to explicitly set to `null`? this field should be `NULLABLE` right?

Comment (seanzhougoogle) [reply src/main/java/io/cdap/delta/bigquery/BigQueryEventConsumer.java:1107]: yes, this filed is nullable. This is for the case when this row was previously deleted and the "_is_deleted" column was set to "true" and now a new insert is to insert the same row , we need to reset "_is_deleted" back to null.

Comment (sagarkapare) [reply src/main/java/io/cdap/delta/bigquery/BigQueryEventConsumer.java:1107]: I see. Thanks for the explanation! Can we add comment describing why we need to set `IS_DELETED` explicitly to `null`?

**Review by sagarkapare — APPROVED** [review]

File: src/main/java/io/cdap/delta/bigquery/BigQueryEventConsumer.java (hunk 1 above)
Comment (seanzhougoogle) [reply src/main/java/io/cdap/delta/bigquery/BigQueryEventConsumer.java:1107]: done :)

**Pull request merged**